cli/coda.ts

```diff
@@ -72,6 +73,24 @@ if (require.main === module) {
       describe: 'Initialize an empty Pack',
       handler: handleInit,
     })
+    .command({
+      command: 'clone <packIdOrUrl>',
+      describe: 'Clone an existing web pack',
+      builder: {
+        // NOTE: yargs doesn't like "version" as a flag name since that's reserved.
+        // Seems like you can override that behavior, but "packVersion" seems fine.
+        packVersion: {
```

Comment:
What's the use case for pulling the non-latest version? Seems like you might run into conflicts if you tried to then upload a new build based on old version.